example.rb

```diff
@@ -75,32 +80,32 @@ def run_sample
         file.write("Hello, World!")
         file.close()
    
-        puts "Temp file = " + full_path_to_file
-        puts "\nUploading to Blob storage as blob" + local_file_name
+        puts "\nCreated a temp file: " + full_path_to_file
+        puts "\nUploading to Blob storage as blob: " + local_file_name
 
         # Upload the created file using local_file_name for the blob name
-        blob_service.create_block_blob(container.name, local_file_name, full_path_to_file)
+        blob_client.create_block_blob(container.name, local_file_name, full_path_to_file)
 
         # List the blobs in the container
-        puts "\n List blobs in the container"
-        blobs = blob_service.list_blobs(container_name)
+        puts "\nList blobs in the container"
+        blobs = blob_client.list_blobs(container_name)
```

Comment:
It would be best if you can show the users how to use `continuation_token` here because we get a lot of customers that report this kind of 'wrong' behaviors for `list_blobs` where the returned object was not in right count of what is actually in the container, but a continuation token is returned and they just ignored..